demisto_sdk/commands/init/initiator.py

```diff
@@ -51,11 +55,13 @@ class Initiator:
            integration (bool): Indicates whether to create an integration.
            script (bool): Indicates whether to create a script.
            full_output_path (str): The full path to the newly created pack/integration/script
+           contribution (str|Nonetype): The path to a contribution zip file
+           description (str): Description to attach to a converted contribution pack
     """
 
     def __init__(self, output: str, name: str = '', id: str = '', integration: bool = False, script: bool = False,
                  pack: bool = False, demisto_mock: bool = False, common_server: bool = False,
-                 contribution: Union[str] = None):
+                 contribution: Union[str] = None, description: str = ''):
```

Comment:
why isnt the desc default `None` ?

Reply:
I want the default description field value to be an empty string rather than the value we set it to in the `create_metadata` method which is `'## FILL MANDATORY FIELD ##'`.